### DISCUSSION TITLE
Ideations

### DISCUSSION THREAD

**Discussion by kettek** [ketMix/magnet#1] [Ideas]
This here be general ideas towards the concept of "magnet".

**Comment by kettek** [comment]
:magnet:

**Comment by liqMix** [comment]
My brainstorming is going in two directions, so I'll be posting a few ideas using these approaches:
- Literal magnet, gameplay revolving around an attraction/pulling mechanic
- Using a play on the word magnet (MagNet, MageNet, Magenta, etc.)

Also these ideas will range from plausible to ridiculous in the hopes of spurring fluid creativity.

**Reply by kettek** [reply]
I do like the idea of using magnetism as a gameplay mechanic.

I also like the idea of a "MagNet" word play. Could be used in conjunction somehow. Perhaps with the tower defense style, it could be how one refers to ones magnetic defense network? Dunno.

**Reply by liqMix** [reply]
Yeaaaa, even if we do go with literal interpretation we can use wordplay for title or some in game description of a system

**Comment by kettek** [comment]
Because of the silly spinning magnet I put in the repo, the thought of a tower defense type of game comes to mind. Could be some interesting mechanics there. Perhaps different enemies could have a positive or negative attraction that you would have to build appropriate defenses against? Targeted item absorption using a magnet beam, whilst firing weapons uses a repulsion force?

**Reply by kettek** [reply]
The bad influence of Factorio also makes me think that it'd be neat to have to power one's turrets and manage upgrading/placing more power sources as needed. Might be too complex in scope unless we keep it fairly simple.

**Reply by liqMix** [reply]
Ooh, that sounds neat. I like the idea of supplying then power through constructed power sources and managing that aspect.
Perhaps there is something interesting we can do with terrain or the environment or something like traps in order to really capitalize on applying attractive/repulsive forces to incoming enemies/items.

**Comment by liqMix** [comment]
A simple circle play area where you control a magnet that rotates around it and attracts/repels various entities within the circle.

**Comment by liqMix** [comment]
Magen't, a game about someone in a magical world who has no magic powers but has a really strong magnet.
MagNet, a shooting game where your gun is actually a computer you can upgrade like you would PC components and you have to download your ammo from the MagNet.

**Comment by liqMix** [comment]
A top-down shooting game where you only have a magnet but luckily all the projectiles shot at you are affected by it. If you aim towards the projectiles you can 'aggregate' them on the magnet then reverse the polarity to fire them.

**Reply by kettek** [reply]
I quite like this concept. In conjunction with MagNet, as well as a potential tower defense style game, it might be quite neat. The concepts are similar enough that I suspect it wouldn't be overkill.

**Comment by liqMix** [comment]
ManGet, you're a very attractive person that needs to find a man to wed

**Comment by liqMix** [comment]
A magnet dating simulator where you woo different magnets

**Comment by liqMix** [comment]
Magnet Carta, you must limit the king's powers by entrapping him through the use of cleverly placed magnets
